### PR TITLE
NS-236/documentation: added README.md

### PR DESCRIPTION
**Pull request by net-illia-stetsenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Net Agent
+
+[![Crates.io](https://img.shields.io/crates/v/net-agent.svg)](https://crates.io/crates/net-agent)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+`net-agent` is a binary crate that allows you to capture network traffic and load it to [my.netstalker.io](https://my.netstalker.io) for further analysis.
```
Comment (net-yehor-tretiakov) [inline README.md]: Net-Agent (Net-agent) is a standalone bin crate (app?).
It is not connected to [my.netstalker.io](https://my.netstalker.io) anyhow.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+- Install Xcode Command Line Tools::
+    ```shell
+    xcode-select --install
+    ```
+- Install libpcap using Homebrew::
+    ```shell
+    brew install libpcap
+    ```
+
```
Comment (net-yehor-tretiakov) [inline README.md]: I would like you to be consistent with style. If you leave empty line somewhere, please think if we place such symbols (new line) in same circumstances at other places.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+```shell
+net-agent [OPTIONS]
+```
+
+The available options are:
+
+- `-D, --device-name <DEVICE_NAME>`: Specify the network device to capture traffic from.
+- `-N, --number-packages <NUMBER_PACKAGES>`: Specify the number of packages to capture.
+- `-B, --buffer-size <BUFFER_SIZE>`: Specify the buffer size for capturing packages.
+- `O, --output-directory <OUTPUT_DIRECTORY>`: Specify the output directory for captured traffic (default: "output").
```
Comment (net-yehor-tretiakov) [inline README.md]: Same thing here with `'O'`. I feel like it should be `'-O'`.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+## Configuration
+
+```toml
+device_name = "eth0"
+number_packages = 1000
+buffer_size = 2048
+output_directory = "captured_traffic"
+```
+
+To use the configuration file, pass the `-C`, `--config-file` option with the path to the file.
```
Comment (net-yehor-tretiakov) [inline README.md]: I will like it more, if we place *things at line 75* before config code. And I would love to see template, not concrete examples.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+## Troubleshooting
+
+On linux to run the application with `sudo` it usually needs to run
+```shell
+sudo cp ~/.cargo/bin/net-agent /usr/local/bin/
```
Comment (net-yehor-tretiakov) [inline README.md:99]: Idk, mb it should be like 
```shell
sudo cp ~/.cargo/bin/net-agent
sudo cp /usr/local/bin/

File: README.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+## Contributing
+
+Contributions to `net-agent` are welcome! If you find any issues or have suggestions for improvements, please open an issue on the GitHub repository.
+If you want to contribute code to the project, please follow these steps:
+
+1. Fork the repository and create a new branch for your feature or bug fix.
```
Comment (net-yehor-tretiakov) [inline README.md]: It should deff not be a fork

File: README.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+## Troubleshooting
+
+On linux to run the application with `sudo` it usually needs to run
+```shell
+sudo cp ~/.cargo/bin/net-agent /usr/local/bin/
+```
+
+## Contributing
+
+Contributions to `net-agent` are welcome! If you find any issues or have suggestions for improvements, please open an issue on the GitHub repository.
```
Comment (net-yehor-tretiakov) [inline README.md]: "All the contributions are welcome!" is a better start, to my taste.

**Review by net-yehor-tretiakov — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
+# Net Agent
+
+[![Crates.io](https://img.shields.io/crates/v/net-agent.svg)](https://crates.io/crates/net-agent)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+Net-agent is a stand-alone binary crate within the Net Stalker project that allows you to capture network traffic, which can then be uploaded to [my.netstalker.io](https://my.netstalker.io/) for further analysis.
```
Comment (net-yehor-tretiakov) [inline README.md]: I suggest, we need to swap "can", and "then"

**Review by net-yehor-tretiakov — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (net-illia-stetsenko) [reply README.md]: Done

**Pull request merged**